### PR TITLE
chore(ci-cd-workflows): GitHub Actions snippets

### PR DESCRIPTION
**Pull request by 0lmi**
Changelog: Title
Ticket: MEN-5888
Signed-off-by: Alex Miliukov <oleksandr.miliukov@northern.tech>

# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [ ] Make sure that all commits follow the conventional commit [specification](https://www.github.com/mendersoftware/mendertesting/commitlint/grammar.md) for the Mender project.

The majority of our contributions are fixes, which means your commit should have
the form below:

```
fix: <SHORT DESCRIPTION OF FIX>

<OPTIONAL LONGER DESCRIPTION>

Changelog: <USER-FRIENDLY-CHANGE-DESCRIPTION> or <None>
Ticket: <TICKET NUMBER> or <None>
```

- [ ] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: GitHab Actions
```
Comment (MaciejTe) [inline 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md:2]: typo: `GitHab`

Comment (0lmi) [reply 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md:2]: GitHub's CI system is called `GitHub Actions`, so, at least from my side it was intentional to call it like that.

File: 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+taxonomy:
+    category: reference
+---
+
+Pre-built container image `mendersoftware/mender-ci-tools` with pre-installed `mender-cli` and `mender-artifact` tools is provided. The image can be used for building Mender artifacts from GitHub Actions jobs and (optionally) uploading them to a Mender server.
```
Comment (TheYoctoJester) [inline 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md:7]:
```suggestion
A pre-built container image `mendersoftware/mender-ci-tools` including the `mender-cli` and `mender-artifact` tools is provided. The image can be used for building Mender artifacts from GitHub Actions jobs and (optionally) uploading them to a Mender server.
```

File: 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## GitHub Actions
+
+!!!!! Only modern Mender backend version with [Personal Access Tokens](../../01.Using-the-apis/docs.md#personal-access-tokens) functionality in place is supported by the actions
```
Comment (TheYoctoJester) [inline 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md:11]:
```suggestion
!!!!! Only modern Mender backend versions with [Personal Access Tokens](../../01.Using-the-apis/docs.md#personal-access-tokens) functionality in place are supported by the actions
```

Do we have a changelog that we can link to, starting which version this applies?

Comment (mzedel) [reply 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md:11]: https://mender.io/blog/mender-3-4-on-prem-release-integrate-mender-with-aws-iot-core-and-other-backend-services has a note on personal access tokens 👌

File: 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+### Setup
+The actions requires the following Secret is set in a repository Settings:
```
Comment (TheYoctoJester) [inline 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md:14]:
```suggestion
The actions require the following secret is set in a repository settings:
```

File: 08.Server-integration/08.CI-CD-workflows/docs.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
+title: CI/CD workflows
+taxonomy:
+    category: docs
+---
+
+Mender team provides templates for different CI/CD systems for the most common use cases. Using the templates it's possible to create production grade CI/CD pipelines to build and deploy Mender artifacts to a device fleets, managed by hosted Mender server.
```
Comment (TheYoctoJester) [inline 08.Server-integration/08.CI-CD-workflows/docs.md:7]:
```suggestion
The Mender team provides templates for different CI/CD systems that cover the most common use cases. Using the templates makes it possible to create production grade CI/CD pipelines to build and deploy Mender artifacts to a device fleets, managed by hosted Mender server.
```

File: 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md (hunk 1 above)
Comment (lluiscampos) [inline 08.Server-integration/08.CI-CD-workflows/01.GitHub-Actions/docs.md:2]: @0lmi What Maciej means is:
```suggestion
title: GitHub Actions
```

**Pull request closed without merging**